tools/tests/README.md

```diff
@@ -33,25 +64,28 @@ property in the configuration.
 
 karma.conf.js:
 ~~~~{.js}
- files: [
-      { pattern: '**/**/*.css', included: true, type: 'css' },
-      { pattern: 'lib/components.development.js', served: true },
-      { pattern: 'actions.js', served: true },
-      '**/scrollable-container/*.js',
-      '**/checkbox/**.js',
-      '**/dropdown/*.js',
-      '**/lib/*.js',
-      '**/menu/*.js',
-      // '**/modal/*.js', comment to disable
-      // '**/radial-menu/*.js', disabled test
-      '**/router/*.js',
-      '**/tabs/*.js',
-      '**/progress-bar/*.js',
-      '**/rangeslider/*.js',
-      '**/automatic-grid/*.js',
-      '**/radio-button/*.js',
-      '**/switch/*js'
-    ],
+files: [
+    { pattern: 'node_modules/coherent-gameface-*/**/*.css', included: true, type: 'css' },
+    { pattern: 'node_modules/coherent-gameface-components/umd/*.development.js', served: true },
+    { pattern: 'node_modules/coherent-gameface-*/umd/*.development.js', served: true },
+    { pattern: 'actions.js', served: true },
+    'scrollable-container/*.js',
+    'checkbox/test.js',

```

Comment:
What if we have another test file in the checkbox folder?

Reply:
It should be like the rest and include all files - change to 'checkbox/*.js.